example/test.py

```diff
@@ -421,12 +415,12 @@ def test_expected_processes(self):
              '--concurrency 1'],
         ])))
 
-    def test_amqp_queues_created(self):
+    def test_amqp_queues_created(self, amqp_container):
```

Comment:
I was thinking more about the AMQP container vs the celery container. We're only asking for the AMQP container here, so it's potentially legal for pytest to run this test without the celery container running. In that case, the queues wouldn't have been created by anything.